dist/dashcore-lib.js

```diff
@@ -55788,8 +55788,7 @@ function ProRegTxPayload(options) {
     this.scriptPayout = options.scriptPayout;
     this.inputsHash = options.inputsHash;
     this.payloadSig = options.payloadSig;
-    this.payloadSigSize = this.payloadSig.length * 2
-    this.protocolVersion = options.protocolVersion;
+    this.payloadSig ? this.payloadSigSize = constants.BLS_SIGNATURE_SIZE : this.payloadSigSize = 0;
```

Comment:
Rewrite as `this.payloadSigSize = this.payloadSig ? constants.BLS_SIGNATURE_SIZE : 0`

Reply:
fixed!